frontend/src/app/features/map/layers/RadiusSearchLayer.tsx

```diff
@@ -1,29 +1,50 @@
+import { Circle, useMap, useMapEvents } from 'react-leaflet';
 import { ActiveToolEnum } from '../../../constants/Constant';
-import { CircleLayer } from '../CircleLayer';
+import { useMapCrosshairsCursor } from '../../../hooks/useMapCrossHairCursor';
 import { Site } from '../MapView';
 import { useMapSearchContext } from '../mapSearchContext/MapSearchContext';
+import { LatLngTuple, LeafletMouseEvent } from 'leaflet';
+import { CrosshairsTooltipMarker } from '../CrossHairToolTipMarker';
 
 interface RadiusSearchLayerProps {
-  radius: number;
   onCrossHairClick: () => void;
   sites: Site[];
   setSites: React.Dispatch<React.SetStateAction<Site[]>> | null;
 }
 export function RadiusSearchLayer({
-  radius,
   onCrossHairClick,
-  sites,
-  setSites,
 }: Readonly<RadiusSearchLayerProps>) {
-  const { activeTool } = useMapSearchContext();
+  const { center, radius, onRadiusCrossHairClick, activeTool } =
+    useMapSearchContext();
+  const map = useMap();
+
+  useMapCrosshairsCursor(map);
```

Comment:
This causes the `CrosshairsCursor` to be visible when the map first loads, even without the circle query param.

Try changing it to this:
```suggestion
  useMapCrosshairsCursor(map, activeTool === ActiveToolEnum.radiusSearch);
```